elfloader-tool/include/arch-arm/elfloader.h

```diff
@@ -22,7 +22,22 @@ typedef void (*init_arm_kernel_t)(word_t ui_p_reg_start,
 
 /* Enable the mmu. */
 extern void arm_enable_mmu(void);
+
+/* These functions are very similar however, there are some small differences
+ * between the ARMv8 and legacy implementation.
+ *
+ * New ARMv8 implementation:
+ * - Does the MMU disabling. This is to keep the time spent with MMU off low.
+ * - Is only meant if seL4 runs in EL2.
+ */
+#if defined(CONFIG_ARCH_AARCH64)
+/* Switches MMU-related stuff: pagetables, MAIR & TCR etc. Works also if the MMU
+ * was off initially. EL2 translation regime only.
+ */
+extern void arm_switch_to_hyp_tables(void);
```

Comment:
Perhaps call it `aarch64_switch_to_hyp_tables` if there is no 32-bit version of it.